src/tests/multihost/alltests/test_offline.py

```diff
@@ -54,3 +55,62 @@ def test_0001_bz1416150(self, multihost, backupsssdconf):
             assert 'Backend is offline' in result
         else:
             pytest.fail("Failed to start sssd")
+
+    @pytest.mark.tier1_2
+    def test_0002_bz1928648(self, multihost, backupsssdconf):
+        """
+        :title: clarify which config option applies to each timeout in the logs
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1928648
+        :id: b6c3a1e4-f0ee-11eb-9718-845cf3eff344
+        :steps:
+          1. Login into server running sssd service.
+          2. Configure SSSD with only 1  id_provider.
+          3. Block "id_provider" using below "iptables" command.
+          4. Step 6 should fail and similar messages
+          should be observed in log file (/var/log/sssd/sssd_<domainname>.log).
+          5. The log snip should contain following timeout parameters.
+          - ldap_opt_timeout
+          - ldap_search_timeout
+          - ldap_network_timeout
+          - dns_resolver_timeout
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+          4. Should succeed
+          5. Should succeed
+        """
+        multihost.client[0].run_command("> /var/log/sssd/sssd_example1.log")
+        multihost.client[0].service_sssd('restart')
+        time.sleep(30)
+        it_cat = "cat /var/log/sssd/sssd_example1.log"
+        cat_read = multihost.client[0].run_command(it_cat)
+        assert 'Setting 6 seconds timeout' in cat_read.stdout_text
+        find_id = multihost.client[0].run_command("id foo1@example1")
+        assert find_id.returncode == 0
+        hostname = multihost.master[0].external_hostname
+        block_ip = multihost.client[0].run_command(f'iptables'
+                                                   f' -I OUTPUT '
+                                                   f'-d {hostname}'
+                                                   f' -j DROP')
+        assert block_ip.returncode == 0
+        user = 'foo1@example1'
+        client_hostname = multihost.client[0].sys_hostname
+        client = pexpect_ssh(client_hostname, user, 'Secret123',
+                             debug=False)
+        with pytest.raises(Exception):
+            client.login(login_timeout=5,
+                         sync_multiplier=1,
+                         auto_prompt_reset=False)
+        allow_ip = multihost.client[0].run_command(f'iptables '
+                                                   f'-I OUTPUT '
+                                                   f'-d {hostname} '
+                                                   f'-j ACCEPT')
```

Comment:
The rule has to be deleted in this cleanup. Here a rule is being added which did not exist before.

Reply:
It was "DROP" in line 92 then here it is "ACCEPT" . So clean up is already done with it .

Reply:
[root@clientrel9 ~]# iptables -L
Chain INPUT (policy ACCEPT)
target     prot opt source               destination         

Chain FORWARD (policy ACCEPT)
target     prot opt source               destination         

Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination         
[root@clientrel9 ~]# 


NO role is there at this moment

Reply:
```
[root@ci-vm-10-0-139-74 ~]# iptables -I OUTPUT -d 10.1.1.3 -j DROP

[root@ci-vm-10-0-139-74 ~]# iptables -nL OUTPUT
Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination         
DROP       all  --  0.0.0.0/0            10.1.1.3            

[root@ci-vm-10-0-139-74 ~]# iptables -I OUTPUT -d 10.1.1.3 -j ACCEPT

[root@ci-vm-10-0-139-74 ~]# iptables -nL OUTPUT
Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination         
ACCEPT     all  --  0.0.0.0/0            10.1.1.3            
DROP       all  --  0.0.0.0/0            10.1.1.3            

[root@ci-vm-10-0-139-74 ~]# iptables -D OUTPUT -d 10.1.1.3 -j ACCEPT

[root@ci-vm-10-0-139-74 ~]# iptables -nL OUTPUT
Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination         
DROP       all  --  0.0.0.0/0            10.1.1.3            

[root@ci-vm-10-0-139-74 ~]# iptables -D OUTPUT -d 10.1.1.3 -j DROP

[root@ci-vm-10-0-139-74 ~]# iptables -nL OUTPUT
Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination
```

Reply:
```
[root@vm ~]# iptables -I OUTPUT -d 10.x.x.x -j DROP

[root@vm ~]# iptables -nL OUTPUT
Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination         
DROP       all  --  0.0.0.0/0            10.x.x.x            

[root@vm ~]# iptables -I OUTPUT -d 10.x.x.x -j ACCEPT

[root@vm ~]# iptables -nL OUTPUT
Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination         
ACCEPT     all  --  0.0.0.0/0            10.x.x.x            
DROP       all  --  0.0.0.0/0            10.x.x.x            

[root@vm ~]# iptables -D OUTPUT -d 10.x.x.x -j ACCEPT

[root@vm ~]# iptables -nL OUTPUT
Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination         
DROP       all  --  0.0.0.0/0            10.x.x.x            

[root@vm ~]# iptables -D OUTPUT -d 10.x.x.x -j DROP

[root@vm ~]# iptables -nL OUTPUT
Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination
```

Reply:
Fixed